solidity/dashboard/src/services/rewards.service.js

```diff
@@ -164,18 +164,18 @@ const fetchWithdrawalHistory = async (web3Context) => {
           const {
             transactionHash,
             blockNumber,
-            returnValues: { groupIndex, amount },
+            returnValues: { groupIndex, amount, operator },
           } = event
-          const withdrawnAt = (await eth.getBlock(blockNumber)).timestamp
           const groupPublicKey = await keepRandomBeaconOperatorContract.methods
             .getGroupPublicKey(groupIndex)
             .call()
           return {
             blockNumber,
             groupPublicKey,
-            date: formatDate(withdrawnAt * 1000),
-            amount: utils.fromWei(amount, "ether"),
+            reward: utils.fromWei(amount, "ether"),
             transactionHash,
+            operator,
+            status: "WITHDRAWN",
```

Comment:
if we extract rewards statuses to `constants` then we'll be able to reuse it here

Reply:
seems that we still do not reference constant